### DISCUSSION TITLE
cara membuat group dan menambahkan kontak ke dalam group

### DISCUSSION THREAD

**Discussion by azilin** [kirimwa-id/forum#84] [Saran dan Ide]
mohon petunjuk untuk membuat group dan menambahkan anggota group 
Terimakasih

source : https://developer.kirimwa.id/api/v1

**Comment by rioastamal** [comment]
Saat ini fitur tersebut belum ada, silahkan request di thread "Saran dan Ide" nanti akan dipertimbangkan untuk diimplementasikan.
